extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/devconsole/InfinispanClientsContainer.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.infinispan.client.runtime.devconsole;
+
+import java.lang.annotation.Annotation;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import jakarta.inject.Singleton;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ServerConfiguration;
+
+import io.quarkus.arc.Arc;
+import io.quarkus.arc.InjectableBean;
+import io.quarkus.arc.InstanceHandle;
+import io.quarkus.arc.Unremovable;
+import io.quarkus.arc.profile.IfBuildProfile;
+import io.quarkus.infinispan.client.runtime.InfinispanClientUtil;
+
+@IfBuildProfile("dev")
+@Unremovable
+@Singleton
+public class InfinispanClientsContainer {
```

Comment:
I assume this is for the old Dev UI?

We should really focus on the new Dev UI.

Reply:
will be handled here very shortly @geoand https://github.com/quarkusio/quarkus/issues/32593